src/sweeps/bayes_search.py

```diff
@@ -429,13 +433,19 @@ def _construct_gp_data(
                 metric = worst_metric
             y.append(metric)
             sample_X.append(X_norm)
+        elif run.state in [RunState.running]:
+            # Use metric for gaussian training while running, NOT default functionality
+            if config["metric"].get("impute_while_running"):
+                metric = tryGetMetric()
+                y.append(metric)
```

Comment:
in the running state, do we also want to append to sample_X? Not exactly sure how we use that...